pkg/lastpass/cli.go

```diff
@@ -36,24 +36,24 @@ func VerifyCliExistsOrDie() {
 
 // Login using lastpass-cli
 func Login(username string, password string) error {
-	// echo <PASSWORD> | LPASS_DISABLE_PINENTRY=1 lpass login --trust <USERNAME>
-	out, err := sh.NewSession().SetEnv("LPASS_DISABLE_PINENTRY", "1").Command("echo", password).Command("lpass", "login", "--trust", username).Output()
-	if err != nil || "" == string(out) {
-		// sometimes returns error: "Error: HTTP response code said error" even if the credentials are valid
-		return fmt.Errorf("verify credentials, unable to login: %s", err)
+	_, err := sh.Command("lpass", "status").Output()
```

Comment:
out of curiosity, any performance issues for doing this i.e. skipping the logout?

Reply:
Hi, i didn't spot any performance issues, but in our use-case, we need to sync hundreds of secrets, with the logout activated, we get banned from Lastpass.
We didn't try to Log in at each secret synchronisation, but we'll probably get banned too from Lastpass.

Reply:
Fair enough!